source/wp-content/themes/wporg-developer/scss/main.scss

```diff
@@ -201,6 +201,11 @@
 		}
 
 		.dashicons {
+			-moz-transform: scale(-1, 1);
+			-webkit-transform: scale(-1, 1);
+			-o-transform: scale(-1, 1);
+			-ms-transform: scale(-1, 1);
```

Comment:
Which browsers need these prefixes?

Reply:
https://caniuse.com/?search=transform
There are about 0.91% of users in total globally still using older browsers needing these prefixes, I used to just add these prefixes, but maybe it's not a good practice, should I just remove them?

(Just found that `-o-` is used for Opera 11.5 and no one is using that version anymore, I'm going to remove it first)